examples/core/transform-feedback-instanced/app.js

```diff
@@ -0,0 +1,406 @@
+    const transformFeedbacks = new Array(2);
+
+    const positionBuffer = new Buffer(gl, {
+      data: trianglePositions,
+      size: 2,
+      type: gl.FLOAT,
+      normalized: false,
+      stride: 0,
+      offset: 0,
+      instanced: 0,
+      usage: gl.STATIC_DRAW
+    });
+
+    const colorBuffer = new Buffer(gl, {
+      data: instanceColors,
+      size: 3,
+      type: gl.FLOAT,
+      normalized: false,
+      stride: 0,
+      offset: 0,
+      instanced: 1,
+      usage: gl.STATIC_DRAW
+    });
+
+    for (let va = 0; va < vertexArrays.length; ++va) {
+      const offsetBuffer = new Buffer(gl, {
+        data: instanceOffsets,
+        size: 2,
+        type: gl.FLOAT,
+        normalized: false,
+        stride: 0,
+        offset: 0,
+        instanced: 1,
+        usage: gl.STREAM_COPY
+      });
+
+      const rotationBuffer = new Buffer(gl, {
+        data: instanceRotations,
+        size: 1,
+        type: gl.FLOAT,
+        normalized: false,
+        stride: 0,
+        offset: 0,
+        instanced: 1,
+        usage: gl.STREAM_COPY
+      });
+
+      vertexArrays[va] = new VertexArray(gl, {
+        buffers: {
+          [COLOR_LOCATION]: {
+            buffer: colorBuffer,
+            size: 3,
+            type: gl.FLOAT,
+            normalized: false,
+            stride: 0,
+            offset: 0,
+            instanced: 1
+          },
+          [OFFSET_LOCATION]: {
+            buffer: offsetBuffer,
+            size: 2,
+            type: gl.FLOAT,
+            normalized: false,
+            stride: 0,
+            offset: 0,
+            instanced: 1
+          },
+          [ROTATION_LOCATION]: {
+            buffer: rotationBuffer,
+            size: 1,
+            type: gl.FLOAT,
+            normalized: false,
+            stride: 0,
+            offset: 0,
+            instanced: 1
+          },
+          [POSITION_LOCATION]: {
+            buffer: positionBuffer,
+            size: 2,
+            type: gl.FLOAT,
+            normalized: false,
+            stride: 0,
+            offset: 0,
+            instanced: 0
+          }
+        }
+      });
+
+      transformVertexArrays[va] = new VertexArray(gl, {
+        buffers: {
+          [OFFSET_LOCATION]: {
+            buffer: offsetBuffer,
+            size: 2,
+            type: gl.FLOAT,
+            normalized: false,
+            stride: 0,
+            offset: 0,
+            instanced: 0
+          },
+          [ROTATION_LOCATION]: {
+            buffer: rotationBuffer,
+            size: 1,
+            type: gl.FLOAT,
+            normalized: false,
+            stride: 0,
+            offset: 0,
+            instanced: 0
+          }
+        }
+      });
+
+      /* eslint-disable camelcase  */
+      transformFeedbacks[va] = new TransformFeedback(gl, {
+        buffers: {
+          v_rotation: rotationBuffer,
+          v_offset: offsetBuffer
+        },
+        varyingMap: modelTransform.program.varyingMap
```

Comment:
I wonder if we should have an accessor on `Model`:

```
   get varyingMap() {
       return this.program.varyingMap;
   }
```

Reply:
Added.